### ISSUE TITLE
[Mars] Retrain model from Earth test case with skorch

### ISSUE THREAD

**Issue by Domnyk** [Domnyk/pallas#24]
Assumptions
* Use already trained model from Earth test case which achieved AUC of 0.77 on train set. Model is [here](https://drive.google.com/drive/folders/102DZD08L1PKFOva8RKmvjduxdtRH0RQ6?usp=sharing)
* Perform extra training with cross validation using [skorch](https://github.com/skorch-dev/skorch)

Expectations
* Difficult to tell. AUC on val set might be better since with K-fold cross validation samples from val set will be used to train net. However, since network is already trained I can achieve worse results on test set because net might overfit to train data
